src/main/resources/application.yaml

```diff
@@ -104,6 +104,13 @@ idm:
     ping:
       freshness-interval: 30000
       check-interval: 1000
+    check-role-exists:
+      freshness-interval: 30000
```

Comment:
We should probably keep that "fresh" for longer - maybe an hour.
Also - check less often.